### DISCUSSION TITLE
Does Chatterino2 support combined platform chat and Relay

### DISCUSSION THREAD

**Discussion by bananaman15** [Chatterino/chatterino2#5358] [Q&A] [answered]
Hi
Does Chatterino combine chats from different platforms (Youtube, Twitch, Kick) etc into one chat box and also support overlaying the combined chatbox as an overlay to display on Stream?

Additionally, does it have relay support whereby chats from different platforms can be displayed in the chatbox? For example if a user on Twitch chats, it will be relayed to the chatbox on Youtube for viewers to see.

Thanks

**Comment by Felanbird** [comment, marked as answer]
no to both questions
